src/domain/wallets/index.types.d.ts

```diff
@@ -24,28 +24,28 @@ type BaseWalletTransaction = {
   readonly deprecated: Deprecated
 }
 
-type WalletNameTransaction = BaseWalletTransaction & {
-  readonly initiationVia: "walletname"
+type IntraLedgerTransaction = BaseWalletTransaction & {
+  readonly initiationVia: "walletid"
   readonly settlementVia: "intraledger"
-  readonly recipientId: WalletName
+  readonly recipient: Username
```

Comment:
Small nit, `recipient` feels like it could sometimes refer to an entire `user` object while something like `recipientName` or `recipientUsername` feels clearer as a single property.

I just had this question when I came across it being used as `tx.recipient` in the gql code. Just an optional comment though, we don't necessarily need to change this.

Reply:
Renamed